src/config/section/infra/routers.js

```diff
@@ -102,7 +102,8 @@ export default {
         targetid: {
           value: (record) => { return record.id }
         }
-      }
+      },
+      response: (result) => { return ((result && result.diagnostics && result && result.diagnostics.url && `Please click the link to download the retrieved diagnostics: <p><a href='${result.diagnostics.url}'>${result.diagnostics.url}</a></p>`) || 'Invalid response') }
```

Comment:
@rhtyd can we perform the translation in JS files?
I've to use string key from here, get the translated string and then replace URL. ELse I'll have to add code in Autogen to explicitly address this API response